### PR TITLE
Initial postgres dumper

### PR DESCRIPTION
**Pull request by boekkooi-fresh**
Let's dump directly into postgres for fun an glory

### CODE REVIEW

File: pkg/dumper/postgres/postgres.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+func (m *driver) NewConnection(dsn string, rdr reader.Reader) (dumper.Dumper, error) {
+	conn, err := sql.Open("postgres", dsn)
+	if err != nil {
+		return nil, err
+	}
+
+	return NewDumper(conn, rdr), nil
+}
```
Comment (kieranajp) [inline pkg/dumper/postgres/postgres.go]: gofmt here

File: pkg/dumper/postgres/postgres.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	"strings"
+
+	"github.com/hellofresh/klepto/pkg/dumper"
+	"github.com/hellofresh/klepto/pkg/reader"
+)
+
+type driver struct{}
+
+func (m *driver) IsSupported(dsn string) bool {
+	return strings.HasPrefix(strings.ToLower(dsn), "postgres://")
```
Comment (kieranajp) [inline pkg/dumper/postgres/postgres.go]: This should be reflected in the docs. Also, how we identify mysql (perhaps we should add `mysql://` to that for consistency...)

File: pkg/dumper/postgres/dumper.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+		return 0, err
+	}
+
+	return insertedRows, nil
+}
+
+func (p *pgDumper) insertIntoTable(txn *sql.Tx, tableName string, rowChan <-chan *database.Row) (int64, error) {
+	columns, err := p.reader.GetColumns(tableName)
+	if err != nil {
+		return 0, err
```
Comment (kieranajp) [inline pkg/dumper/postgres/dumper.go]: hmm, I'd just return `inserted` anyway, but I guess it doesn't matter.

File: pkg/dumper/postgres/postgres.go (hunk 1 above)
Comment (boekkooi-fresh) [reply pkg/dumper/postgres/postgres.go]: Done

File: pkg/dumper/postgres/dumper.go (hunk 3 above)
Comment (boekkooi-fresh) [reply pkg/dumper/postgres/dumper.go]: Why nothing was inserted yet

File: .travis.yml [hunk 4]
```diff
@@ -4,6 +4,12 @@ language: go
 go:
   - 1.x
 
+services:
+  - postgresql
```
Comment (italolelis) [inline .travis.yml]: You have docker compose, just user `docker` here!
And on the before_script run `docker-compose up`

File: pkg/dumper/postgres/dumper.go [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+			if err != nil {
+				logger.WithError(err).Error("Failed to dump table")
+				return
+			}
+
+			logger.WithField("rowCount", insertedRows).Info("Dumped table")
+		}(tableName)
+	}
+
+	go func() {
```
Comment (italolelis) [inline pkg/dumper/postgres/dumper.go]: This goroutine is not necessary

**Comment by boekkooi-fresh** [comment]
Closed since this was split into #50, #51, #52, #53, #54

**Pull request closed without merging**